src/integrationtest/java/com/ericsson/ei/utils/AMQPCommunication.java

```diff
@@ -0,0 +1,104 @@
+package com.ericsson.ei.utils;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+
+/**
+ * Basic AMQP communication class.
+ */
+public class AMQPCommunication {
+    ConnectionFactory factory;
+    Connection connection;
+    Channel channel;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AMQPCommunication.class);
+
+    /**
+     * AMQPCommunication constructor.
+     *
+     * @param host
+     *            host name
+     * @param port
+     *            port number
+     * @throws TimeoutException
+     * @throws IOException
+     */
+    public AMQPCommunication(final String host, final int port) throws IOException, TimeoutException {
```

Comment:
I do not see that this constructor is used. If it is not used please remove it. If you want to keep it the call it from the other one instead of having code duplication.

Reply:
I simply removed it. No need to keep it since everything else that was not needed was also removed.